bundles/org.openhab.binding.dolbycp/README.md

```diff
@@ -0,0 +1,91 @@
+# DolbyCP Binding
+
+This binding is intended to connect to a _Dolby Digital Cinema Processor CP750_.
+Support for CP950 may be added in future (if someone who owns one helps to test it).
+
+It uses ASCII commands send over a TCP connection on port 61408.
+Please note that the CP750 only accepts up to 20 simultaneous connection and will discard the oldest connection if a 21st connection is established.
+So be sure to grateful shutdown each connection using the client's close() method, or it's AutoCloseable functionality.
+
+This binding wraps the CP750 Java library from https://github.com/Cybso/cp750-java/.
+
+This project is NOT affiliated with, funded, or in any way associated with Dolby Laboratories, Inc.
+
+## Supported Things
+
+- `cp750` - The Dolby Digital Cinema Processor CP750 Device.
+
+## Thing Configuration
+
+Normally, only the hostname or IP address must be configured.
+
+### `cp750` Thing Configuration
+
+| Name              | Type    | Description                                                  | Default | Required | Advanced |
+|-------------------|---------|--------------------------------------------------------------|---------|----------|----------|
+| hostname          | text    | Hostname or IP address of the device                         | N/A     | yes      | no       |
+| port              | integer | TCP port if different from 61408                             | 61408   | no       | no       |
+| refreshInterval   | integer | Interval the device is polled in seconds                     | 5       | no       | no       |
+| reconnectInterval | integer | Interval a new connection is tried after IO error in seconds | 10      | no       | no       |
+
+## Properties
+
+| Name              | Description                                              |
+|-------------------|----------------------------------------------------------|
+| osversion         | The operating system's version as returned by the device |
```

Comment:
```suggestion
| osVersion         | The operating system's version as returned by the device |
```
https://www.openhab.org/docs/developer/guidelines.html#naming-convention  Thing properties should be camelCase